Files/ViewModels/ColumnsViewModel.cs

```diff
@@ -6,6 +6,7 @@ namespace Files.ViewModels
 {
     public class ColumnsViewModel : ObservableObject
     {
+        [JsonIgnore]
```

Comment:
Does the [JsonIgnore] attribute work if you put it on a private field? Isn't it is supposed to be put on the `public ColumnViewModel IconColumn` property below?

Reply:
Yeah, thanks for catching that